sibylapp/view/feature_contribution.py

```diff
@@ -1,11 +1,34 @@
 import streamlit as st
+from st_aggrid import AgGrid
 
 from sibylapp.compute import contributions
 from sibylapp.compute.context import get_term
+from sibylapp.config import PREDICTION_TYPE, PredType
 from sibylapp.view.utils import filtering, helpers
+from sibylapp.view.utils.helpers import NEG_EM, POS_EM
 
 
 def show_sorted_contributions(to_show, sort_by):
+    st.write("LEGEND:")
+    modelPred = get_term("Prediction")
+    posChange = ""
+    negChange = ""
+    # TODO: do i need the legend for categorical?
```

Comment:
Let's ignore categorical for now (so you can delete the "else" below). That would get quite complicated, given that we still only offer contributions in two directions. In the event of a categorical model prediction, please go ahead and use the "numeric" situation (everything in the app is currently built for a univariate model prediction, so even if the model prediction is being parsed as categorical, there will still be an underlying numeric output).